pool/app/gtx_config/gtx_config.mli

```diff
@@ -0,0 +1,41 @@
+module Id : sig
+  include Pool_model.Base.IdSig
+end
+
+module ApiKey : sig
+  include Pool_model.Base.StringSig
+end
+
+module Sender : sig
+  include Pool_model.Base.StringSig
+end
+
+type t =
+  { id : Id.t
+  ; api_key : ApiKey.t
+  ; sender : Sender.t
+  ; created_at : Pool_common.CreatedAt.t
+  ; updated_at : Pool_common.UpdatedAt.t
+  }
```

Comment:
Should the type fields be exposed? or just `type t`

Reply:
IMO its simpler than using a function for each field